commands.go

```diff
@@ -327,12 +331,16 @@ func cdCMD(rest string, u *user) {
 		}
 	}
 	if strings.HasPrefix(rest, "#") {
-		u.room.broadcast(u.name, "cd "+rest)
-		if v, ok := rooms[rest]; ok {
+		newRoom := rest
```

Comment:
Why did you use a new variable here?

Reply:
Because the content of newRoom could be the raw value of `rest` or a shorten version depending of this size. It felt cleaner to use a new variable rather that having to mutate the value of `rest`.

Reply:
Golang is pass-by-value though